Source/VolunteerReporting/Events.External/DefaultAutomaticReplyDefined.cs

```diff
@@ -0,0 +1,15 @@
+using doLittle.Events;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Events.External
+{
+    public class DefaultAutomaticReplyDefined : IEvent
+    {
+        public Guid Id { get; set; }
+        public DefaultAutomaticReplyType Type { get; set; }
```

Comment:
Use an integer as we discussed in another issue that I can't find right now. For make versioning easier.

Reply:
So, I guess an integer in the event class, and an enum in concept, for readability in code?

Reply:
Yup - makes sense